tests/integration/model/test_models.py

```diff
@@ -0,0 +1,126 @@
+"""Test API.
+
+Tests the models using API. The weight paths from the trained models are used for the rest of the tests.
+"""
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+from pathlib import Path
+
+import pytest
+from lightning.pytorch.callbacks import ModelCheckpoint
+
+from anomalib.data import AnomalibDataModule, MVTec, TaskType, UCSDped
+from anomalib.engine import Engine
+from anomalib.models import AnomalyModule, get_available_models, get_model
+
+
+def models() -> list[str]:
```

Comment:
Would this be needed elsewhere? Maybe placed into a conftest as a fixture in the future?

Reply:
Once we fix `ai_vad` then we can get rid of this fixture and call `get_avaliable_models` here. Plus this is the only test that uses all the models. The others use only Padim or a random model

Reply:
ah right. Thanks for the clarification!